src/__tests__/interfaces.ts

```diff
@@ -0,0 +1,11 @@
+interface StateObject {
+  year? : number;
+  username? : string;
+  test? : string;
+  data?: {
+      testingLibrary? : string;
+  };
+  numbers ? : Array<number>;
+}
+
+export default StateObject;
```

Comment:
Can you do `export default interface StateObject` to keep it consistent with the other one? 🤗